Frameworks/UIKit/UIApplication.mm

```diff
@@ -1258,301 +992,6 @@ - (void)_sendHighMemoryWarning {
-
-    [_blankView setFrame:startRect];
-    [popupRotationLayer addSubview:_blankView];
-
-    [UIView beginAnimations:@"ResizeAnimation" context:nil];
-
-    if (newHeight <= 0) {
-        //  Keyboard is being hidden
-        [UIView setAnimationDelegate:self];
-        [UIView setAnimationDidStopSelector:@selector(_keyboardDismissed)];
-    }
-    [_blankView setFrame:endRect];
-    [UIView commitAnimations];
-
-    [popupWindow bringSubviewToFront:popupRotationLayer];
-
-    CGRect keyboardBounds;
-    keyboardBounds.origin.x = 0.0f;
-    keyboardBounds.origin.y = 0.0f;
-    keyboardBounds.size = endRect.size;
-
-    CGPoint centerBegin, centerEnd;
-
-    centerBegin.x = mappedStart.origin.x + mappedStart.size.width / 2.0f;
-    centerBegin.y = mappedStart.origin.y + mappedStart.size.height / 2.0f;
-
-    centerEnd.x = mappedEnd.origin.x + mappedEnd.size.width / 2.0f;
-    centerEnd.y = mappedEnd.origin.y + mappedEnd.size.height / 2.0f;
-
-    // Fire the notification:
-    id keys[7] =
-    { @"UIKeyboardFrameEndUserInfoKey",
-      @"UIKeyboardFrameBeginUserInfoKey",
-      @"UIKeyboardAnimationDurationUserInfoKey",
-      @"UIKeyboardAnimationCurveUserInfoKey",
-      @"UIKeyboardBoundsUserInfoKey",
-      @"UIKeyboardCenterBeginUserInfoKey",
-      @"UIKeyboardCenterEndUserInfoKey",
-    };
-    id values[7] = {
-        [NSValue valueWithCGRect:mappedEnd],  [NSValue valueWithCGRect:mappedStart],    [NSNumber numberWithDouble:0.25],
-        [NSNumber numberWithInt:0],           [NSValue valueWithCGRect:keyboardBounds], [NSValue valueWithCGPoint:centerBegin],
-        [NSValue valueWithCGPoint:centerEnd],
-    };
-
-    id dict = [NSDictionary dictionaryWithObjects:values forKeys:keys count:7];
-
-    static float oldHeight = 0;
-
-    if (newHeight > 0) {
-        if (!blankViewUp || forceKeyboardAppearance) {
-            [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardWillShowNotification" object:nil userInfo:dict];
-            [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardDidShowNotification" object:nil userInfo:dict];
-        }
-        if (!blankViewUp) {
-        } else {
-            if (oldHeight != newHeight) {
-                [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardWillChangeFrameNotification"
-                                                                    object:nil
-                                                                  userInfo:dict];
-                [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardDidChangeFrameNotification"
-                                                                    object:nil
-                                                                  userInfo:dict];
-            }
-        }
-    } else {
-        //  Keyboard is being hidden
-        [[NSNotificationCenter defaultCenter] postNotificationName:@"UIKeyboardWillHideNotification" object:nil userInfo:dict];
```

Comment:
This is all dead code - it never worked or provided any value.  Keyboard resize works in a totally different (out-of-proc) manner on Windows.  I'll file a bug to make sure we're tracking plumbing through the UWP keyboard resize events though.  

---
In reply to: [110294830](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r110294830) [](ancestors = 110294830)